tests/model_serving/model_runtime/vllm_omni/modelcar/test_vllm_omni_modelcar.py

```
@@ -0,0 +1,260 @@
+        base_url = get_exposed_isvc_url(isvc=vllm_omni_modelcar_inference_service)
+        speech_url = f"{base_url}{AUDIO_SPEECH_ENDPOINT}"
+        model_name = vllm_omni_modelcar_inference_service.instance.metadata.name
+        payload: dict[str, str] = {
+            "model": model_name,
+            "input": TTS_INPUT_TEXT,
+            "voice": TTS_VOICE,
+            "response_format": "wav",
+        }
+
+        LOGGER.info(event="POST /v1/audio/speech", url=speech_url, model=model_name, voice=TTS_VOICE)
+        response = requests.post(url=speech_url, json=payload, verify=False, timeout=120)
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)